docusaurus/docs/adding-bootstrap.md

```diff
@@ -32,6 +32,12 @@ import 'bootstrap/dist/css/bootstrap.css';
 Sometimes you might need to tweak the visual styles of Bootstrap (or equivalent package).<br>
 As of `react-scripts@2.0.0` you can import `.scss` files. This makes it possible to use a package's built-in Sass variables for global style preferences.
 
+To enable `scss` in Create React App you will need to install `node-scss`:
+
+```sh
+yarn add node-scss
```

Comment:
I think you mean `node-sass`? Also please show the instructions for npm and yarn, similar to the bootstrap installation instructions.